logistics/scripts/modules/path-resolver.psm1

```diff
@@ -123,7 +123,7 @@ function Get-RepositoryNameFromPath ($path) {
     }
     #TODO:Enhance this logic so that it handles "C:/projects/Ed-Fi-Core" (no trailing slash) as an input
     foreach ($repoName in $repositoryNames) {
-        $repoRoot = "$(Get-RepositoryRoot $repoName)\"
+        $repoRoot = "$(Get-RepositoryRoot $repoName)$([IO.Path]::DirectorySeparatorChar)"
```

Comment:
This is something that I should have caught when I reviewed this file in [ODS-5370](https://tracker.ed-fi.org/browse/ODS-5370); fixing it now.

Reply:
Good catch